### PR TITLE
Create a new plugin submission template

### PR DESCRIPTION
**Pull request by Hecter94**
This allows anyone to easily submit a request to add a new plugin to the archive without requiring a PR. 
The template asks for the following information:
author
website
directlink
category
status
description
long description

When submitted, the issue will automatically be tagged and assigned, allowing manual addition of the plugin to the archive, at which point the issue can be completed.

### CODE REVIEW

**Comment by RisingLeaf** [comment]
A short description would also be nice?

File: .github/ISSUE_TEMPLATE/plugin-submission.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Status
+      description: Please enter the current status of the plugin, whether it is complete, in progress, abandoned, etc.
+  - type: textarea
+    attributes:
+      label: Description
+      description: Please include a short description of the plugin.
```
Comment (Hecter94) [inline .github/ISSUE_TEMPLATE/plugin-submission.yaml:61]: > A short description would also be nice?

There is a "Description" textarea that someone submitting a plugin could use to describe the plugin.
Do you mean including both a "short description" and a "long description"? Where would this short description be shown that the long description isn't?

Comment (RisingLeaf) [reply .github/ISSUE_TEMPLATE/plugin-submission.yaml:61]: referring to this: https://github.com/endless-sky/endless-sky/discussions/9058
a short description could be a tooltip

Comment (RisingLeaf) [reply .github/ISSUE_TEMPLATE/plugin-submission.yaml:61]: and yes both long and short

Comment (RisingLeaf) [reply .github/ISSUE_TEMPLATE/plugin-submission.yaml:61]: thank you

**Comment by zuckung** [comment]
And the plugin name is missing

**Comment by Hecter94** [comment]
> And the plugin name is missing

Wouldn't we be able to get that either from the uploaded zip or the directlink?

**Comment by zuckung** [comment]
As soon as a direct link is found in a txt file, the script renames the downloaded zip, and the first in-zip-folder to the name of the txt file. In some cases if no data folder is found, it goes 1 folder deeper. So there is a lot of renaming based on the txt file name.

We have very different repos as source for updates. Tbh only my own plugins have the correct plugin name as zip name. All others have a generated full repo zip which is often named something else than the plugin  name. I.e "xyz plugin" vs "xyz" or "mod" or "-" instead of space, etc. So the txt is the only stable source of the plugin name.

**Comment by zuckung** [comment]
The workflow and script to process this issue template needs some changes.
An input on top named Pluginname and the deletion of the long description input.

Also when copying this template for testing I got some errors with body part 5 and 6. Can't really remember what it was in detail. But was easy to fix by copying the parameters of the earlier bodies.

Here is my template I used for testing, so you can compare.
https://github.com/zuckung/test3/blob/main/.github/ISSUE_TEMPLATE/plugin-submission.yaml

File: .github/ISSUE_TEMPLATE/plugin-submission.yaml (hunk 1 above)
Comment (Hecter94) [reply .github/ISSUE_TEMPLATE/plugin-submission.yaml]: We're now pulling the long description from the plugin readme. The existing description field will be a short description.

**Pull request merged**